assets/scss/layouts/header/_header.scss

```diff
@@ -65,11 +68,14 @@
     > a {
         @include clearfix;
         color: $storeName-color;
-        display: inline-block;
-        padding: $header-logo-paddingVertical 0;
+        height: inherit;
+        display: flex;
+        align-items: center;
+        margin-right: auto;
+        margin-left: auto;
```

Comment:
can we replace it to
`margin: 0 auto;` ?

Reply:
Done